T/Torch_jll/Compat.toml

```diff
@@ -1,2 +1,13 @@
-[1]
+["1-1.4"]
 julia = "1"
+
+["1.10-1"]
+CPUInfo_jll = "0.0.20201217"
+CUDNN_jll = "8"
+Gloo_jll = "0.0.20210521"
+JLLWrappers = "1.2.0-1"
+PThreadPool_jll = "0.0.20210414"
+SLEEF_jll = "3"
+XNNPACK_jll = "0.0.20210622"
+julia = "1.6.0-1"
+protoc_jll = "3.13.0-3.13"
```

Comment:
Had to limit compat of `protoc_jll`, see https://github.com/JuliaPackaging/Yggdrasil/pull/5551/files#r979472676 and https://github.com/JuliaPackaging/Yggdrasil/pull/5576.

[noblock]